sources/packages/api/src-sql/SINValidations/Create-sin-validations.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE IF NOT EXISTS sims.sin_validations (
+  id SERIAL PRIMARY KEY,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  given_name_sent VARCHAR(50),
+  surname_sent VARCHAR(50),
+  dob_sent Date,
+  valid_sin boolean,
+  request_status_code CHAR(2),
+  match_status_code CHAR(2),
+  sin_match_status_code CHAR(2),
+  surname_match_status_code CHAR(2),
+  given_name_match_status_code CHAR(2),
+  dob_match_status_code CHAR(2),
+  user_id INT REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
Usually upon user removal with are setting null instead of DELETE the record.

Reply:
y do we need the records, even after user is removed? We cannot track or audit without any user information, except guessing the user with the surname and given name sent :)

Reply:
We need to keep the sin validation status of the student. We are not removing the student, for instance, upon user removal.

Reply:
In general, I mentioned to @abschwenker that I would like to avoid all CASCADING operations, it is usually dangerous while on PROD. I would preferer that, if there is a need to remove a record, the dev/DBA need explicitly remove the records.